handlebars/src/test/java/io/knotx/te/handlebars/HandlebarsTemplateEngineTest.java

```diff
@@ -0,0 +1,74 @@
+package io.knotx.te.handlebars;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalToIgnoringWhiteSpace;
+import static org.mockito.Mockito.when;
+
+import io.knotx.dataobjects.Fragment;
+import io.knotx.junit5.util.FileReader;
+import io.vertx.core.json.JsonObject;
+import io.vertx.reactivex.core.Vertx;
+import java.io.IOException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class HandlebarsTemplateEngineTest {
```

Comment:
Please add the scenarios:
- incorrect syntax,
- empty fragment

Reply:
Not sure what  you mean by incorrect syntax.
There is already test for spotting delimiters that are not HBS ones (in `process_whenCustomDelimiter_expectMarkup`).
I've added one more test `process_whenUndefinedHbsHelperSpotted_expectMarkup` where unknown hbs helper is used.

Reply:
ok